### ISSUE TITLE
When using Voiceover, the close button next to the search input field cannot be pressed.

### ISSUE THREAD

**Issue by itsumoonazicode** [GoogleChrome/developer.chrome.com#2294]
⚠️ This is for issues with the https://developer.chrome.com site, not Chromium itself.
If you want to file a bug with Chromium (the open-source project behind Google Chrome and other browsers), please use https://crbug.com.

Translated from Japanese to English using DeepL. It may be difficult to read, so I apologize in advance.

**Describe the bug**
When I use Voiceover on the iPad OS and focus on the search input field, I cannot focus on the close button. Therefore, once I open the search input field, I am unable to view the content I was just viewing.

**To Reproduce**
1. Start Voiceover.
2. Open Safari
3. Go to ['https://developer.chrome.com/'](https://developer.chrome.com/)
4. Click on 'search button'

**Expected behavior**
The close button can be pressed.

**Screenshots**
![image](https://user-images.githubusercontent.com/28680683/158304092-769d5892-8dfc-434a-88b6-444c2c96aaa0.png)

**Smartphone (please complete the following information):**
 - Device: iPad 7th generation
 - OS: iPad OS 15.4
 - Browser: Safari
 - Version: 15.4

**Additional context**
It appears that the skip to content link is covering the close button, so the button cannot be pressed.

**Comment by matthiasrohmer** [comment]
Won't fix, due to the pending move to the new CMS.
